text/0693-property-injection.md

```diff
@@ -0,0 +1,526 @@
+# Property Injection Implementation of Blueprints
+
+* **Original Author(s):**: @pcheungamz
+* **Tracking Issue**: [#693](https://github.com/aws/aws-cdk-rfcs/issues/693)
+* **API Bar Raiser**: @rix0rrr
+
+`Property injection` is a new mechanism that makes it possible to control the properties that are used to instantiate a construct via an out-of-band mechanism.
+Constructs explicitly opt in to being injectable; all L2 constructs in the AWS CDK standard library will be property injectable.
+
+Property injection allows the implementation of `Blueprints`:
+a collection of property injectors vended inside a large organization to make it easy for application builders to apply
+organizational best practices when configuring constructs.
+"Blueprints" is a branding term, and they do not imply or require any new technical capability besides property injection.
+Blueprints by themselves are not a compliance enforcement mechanism;
+instead they are a mechanism to make it easier for developers to hit compliance targets that are already being enforced via other means.
+
+**Why do we need Blueprints?**
```

Comment:
How do Property Injectors interact with Aspects? When should I use property injectors to modify resources and when aspects? Based on the RFC so far I don't get yet why I should use injectors at all instead of just modifying the resources via Aspects.